.gitignore

```diff
@@ -163,4 +163,6 @@ full_config*.yaml
 # yaml / json schemas
 schema.json
 
-wandb
\ No newline at end of file
+wandb
+
+local_search_configs
```

Comment:
How about local_configs / custom_configs?

Reply:
this is a folder that already exists from previous PR, for configs from local search, I think the name is descriptive enough

Reply:
yup, at first i thought it is for your local configs, but then relized it is for local_search :D